src/gl/context.cpp

```diff
@@ -364,21 +449,21 @@ void Context::setCameraMVP() {
 
 void Context::drawModel() {
   glClearColor(0.0f, 0.0f, 0.0f, 1.0f);
-  glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
+  glClear(GL_DEPTH_BUFFER_BIT);
   glDepthFunc(GL_LESS);
   model.drawAll();
   glDepthFunc(GL_EQUAL);
 }
 
 void Context::drawExcept(const std::vector<SurfaceBuffer> &hiddenSurfaces) {
   glClearColor(0.0f, 0.0f, 0.0f, 1.0f);
-  glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
```

Comment:
@mbadams5 Why did this change? This is causing the strangeness in the renderings.

Reply:
@nealkruis I don't remember why I made the change. I wasn't really concerned with the visible renderings (since EnergyPlus is all offscreen rendering) so I didn't check the awning code.